universe/UniverseGenerator.cpp

```diff
@@ -5,6 +5,7 @@
 #include "../util/i18n.h"
 #include "../util/Logger.h"
 #include "../parse/Parse.h"
+#include "../python/server/PythonServerFramework.h"
```

Comment:
What exactly is that needed for? If I comment out that line, FO builds just fine (besides being the kind of circular dependency I think should be avoided).

So, unless you object, I'm going to remove that line.